web/about.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>About | Rerum Playground</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+        <style>
+            .header {
+                padding: 15px;
+                text-align: left;
+                z-index: 1;
+                background: #222222;
+            }
+
+            .header img {
+                display: block;
+                margin-left: auto;
+                margin-right: auto;
+                height: 80px;
+            }
+
```

Comment:
Use separate external CSS file and also we should be using same fonts and their sizes across the website.